.gitlab-ci.yml

```diff
@@ -641,6 +641,8 @@ library:ci-fcsl_pcm:
 
 library:ci-fiat_crypto:
   extends: .ci-template-flambda
+  variables:
+    COQEXTRAFLAGS: "-async-proofs off"
```

Comment:
Does that work? In ci-bedrock2.sh we have `COQMF_ARGS='-arg "-async-proofs-tac-j 1"'`